### ISSUE TITLE
Friending or Following another birder

### ISSUE THREAD

**Issue by carzooma** [jefferythewind/b#25]
This might be further down the line but we might want people to friend or follow one another, create flocks?  not sure what you would call it, people might want to follow those near to them or just people who take really good birding pictures.

**Comment by jefferythewind** [comment]
Yeah we might want to think about that a little more.  I have been thinking that the community aspect is super important, this is part of that.  We want to make sure we do that stuff well.
